python/ray/util/sgd/torch/examples/image_models/train.py

```diff
@@ -58,6 +62,42 @@ def data_creator(config):
 
     args = config["args"]
 
+    train_dir = join(args.data, "train")
+    val_dir = join(args.data, "val")
+
+    if args.mock_data:
+        os.makedirs(train_dir, exist_ok=True)
+        os.makedirs(val_dir, exist_ok=True)
+
```

Comment:
Can we put this into a utility file? so that the user reading the code won't need to try to understand what this is doing